src/modules/mod.rs

```diff
@@ -150,3 +151,31 @@ pub fn description(module: &str) -> &'static str {
         _ => "<no description>",
     }
 }
+
+fn undistract_me<'a>(context: &'a Context, module: Option<Module<'a>>) -> Option<Module<'a>> {
+    use ansi_term::{unstyle, ANSIStrings};
+    use notify_rust::{Notification, Timeout};
+
+    if let Some(module) = module.as_ref() {
+        let config: CmdDurationConfig = CmdDurationConfig::try_load(module.config);
+        if config.show_notifications && config.min_time_to_notify as u128 <= context.elapsed()? {
+            let body = format!(
+                "Command execution {}",
+                unstyle(&ANSIStrings(&module.ansi_strings()))
+            );
+
+            let mut notification = Notification::new();
+            notification
+                .summary("Command finished")
+                .body(&body)
+                .icon("bash")
```

Comment:
According to the notify_rust docs `.icon()` needs to be freedesktop.org compliant. Would ["utilities-terminal"](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) be a more standard choice here?

More importantly, does "utilities-terminal" even work, or does it throw up a generic "no icon" icon?

Reply:
Yes, utilities-terminal is a better choice.